customconfig/config.go

```diff
@@ -1,6 +1,7 @@
 package customconfig
 
 type Config struct {
+	TimeoutMinutes int
```

Comment:
A `TimeoutMinutes` would prevent being able to set fractional minute timeouts such as 30 seconds both here because it's an `int` and when used to cast to a `time.Duration` as `time.Duration(0.5) * time.Minute` throws `constant 0.5 truncated to integer`.  Can we use `TimeoutSeconds` instead?